nbs/animation.ipynb

```diff
@@ -41,50 +43,36 @@
         "%%capture\n",
         "#@title Connect to the Stability API\n",
         "\n",
-        "from base64 import b64encode\n",
         "import datetime\n",
         "import os\n",
-        "from pathlib import Path\n",
+        "import panel as pn\n",
+        "import param\n",
         "import shutil\n",
         "import subprocess\n",
         "import sys\n",
         "\n",
+        "from base64 import b64encode\n",
+        "from IPython import display\n",
+        "from pathlib import Path\n",
+        "from PIL import Image\n",
+        "from tqdm import tqdm\n",
+        "\n",
         "\n",
+        "# install Stabiliy SDK for Python\n",
         "path = Path('stability-sdk')\n",
-        "if path.exists():\n",
-        "    shutil.rmtree(path)\n",
-        "    !pip uninstall -y stability-sdk\n",
-        "\n",
-        "!git clone -b dmarx.anima.dev --recurse-submodules https://github.com/Stability-AI/stability-sdk\n",
-        "!touch ./stability-sdk/src/stability_sdk/interfaces/__init__.py\n",
-        "!pip install ./stability-sdk\n",
+        "if not is_colab and os.path.exists(\"../src/stability_sdk\"):    \n",
```

Comment:
should still install sdk if using it locally. if you're thought process here was to facilitate development, we can do an "editable" install with `pip install -e ./stability-sdk` which will respect code changes the same way tinkering with the path variable would.

Reply:
yeah I wanted to make development easier and point directly to source code that is being edited while working on the notebook. doing the pip uninstall and pip install adds extra time while iterating on changes. it's specific to development so I can remove from git version and just paste in while doing dev if you prefer to note have it there.

Reply:
I've removed 👍

Reply:
the pip uninstall and re-install was just cause I was lazy and although it's not as fast as it could be, the package is still pretty lightweight so this uninstall-reinstall procedure doesn't have an especially high overhead (imho). either way, pip install -e I think achieves what you're looking for. only edge case is if you modify dependencies in e.g. setup.py, then you'll need to re-install for that to be respected.